src/opening-period/page/CreateNewOpeningPeriodPage.test.tsx

```diff
@@ -167,7 +166,36 @@ describe(`<CreateNewOpeningPeriodPage />`, () => {
   let testDatePeriod: DatePeriod;
 
   beforeEach(() => {
-    testDatePeriodConfig = datePeriodOptions;
+    testDatePeriodConfig = {
```

Comment:
This change seems unnecessary (rebase?) 🤔

Reply:
Rebase issues should be resolved now 😅